src/main/java/org/dita/dost/ant/InitializeProjectTask.java

```diff
@@ -28,6 +28,11 @@
  * @since 3.5
  */
 public final class InitializeProjectTask extends Task {
+
+    private static ServiceLoader<StoreBuilder> storeBuilderLoader = ServiceLoader.load(StoreBuilder.class);
```

Comment:
Use service loader to load `StoreBuilder` which in turn can be used to create the actual `Store` implementations.